src/lib/styles/tailwind.config.ts

```diff
@@ -3,6 +3,7 @@ import defaultTheme from 'tailwindcss/defaultTheme'
 import type { Config } from 'tailwindcss'
 
 export default {
+  jit: true,
```

Comment:
should we set the `purge` arg too? https://v2.tailwindcss.com/docs/just-in-time-mode#enabling-jit-mode

Reply:
ah i left it there yes, I was trying to do the breakpoint range but it didnt work